mailing_christmas/report/payment_slip.py

```diff
@@ -0,0 +1,260 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Philippe Heer
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+from __future__ import division
+import base64
+import StringIO
+import contextlib
+from collections import namedtuple
+from reportlab.pdfgen.canvas import Canvas
+from reportlab.lib.units import inch
+from openerp import models, api, _
+from openerp.tools.misc import mod10r
+
+FontMeta = namedtuple('FontMeta', ('name', 'size'))
+
+
+class PaymentSlip(models.Model):
+    """From Version 8 payment slip are now a
```

Comment:
You can remove this comment and replace by "Add methods to print Christmas Payment Slip for partners"